GitTfs.VsCommon/TfsHelper.Common.cs

```diff
@@ -657,13 +657,21 @@ private Workspace GetWorkspace(params WorkingFolder[] folders)
             var workspace = VersionControl.CreateWorkspace(GenerateWorkspaceName());
             try
             {
-                foreach (WorkingFolder folder in folders)
-                    workspace.CreateMapping(folder);
+                SetWorkspaceMappingFolders(workspace, folders);
             }
             catch (MappingConflictException e)
             {
-                TryToDeleteWorkspace(workspace);
-                throw new GitTfsException(e.Message).WithRecommendation("Run 'git tfs cleanup-workspaces' to remove the workspace.");
+                try
+                {
+                    foreach (WorkingFolder folder in folders)
+                        CleanupWorkspaces(folder.LocalItem);
+                    SetWorkspaceMappingFolders(workspace, folders);
```

Comment:
That's the goal of this PR to do what the `cleanup-workspaces` do, when setting the workspace mappings has failed before trying again to set the mappings again. It just a (very!) convenient fix to prevent the user (and first of all, me when I develop!!) to see the command fail, have to cleanup and need to launch again its command (sometimes you even have to continue with a different command!) if we can in fact just delete the workspace and continue without problem...

If for some reason, that I can't imagine, that fail again, then at that time git-tfs fail and delete the newly created workspace...